src/web/server/app/utils/mailer.ts

```diff
@@ -0,0 +1,30 @@
+import nodemailer from 'nodemailer';
+import { EMAIL, EMAIL_PASSWORD, REACT_SERVER_IP } from './settings';
+
+export const sendEmail = async (recipient: string, token: string) => {
+  try {
+    const transporter = nodemailer.createTransport({
+      service: 'Gmail',
```

Comment:
This wouldn't work if EMAIL isn't gmail right?

Reply:
Guess not.